kjourney/src/app/band/band.component.html

```diff
@@ -0,0 +1,57 @@
+<router-outlet></router-outlet>
+<header>
+<h1>Executive</h1>
+
+<button class="button1" style="width:15%">View Competencies</button>
+
+<section>
+  <article>
+	<table id="careerTable" style="width:100%; overflow: auto;">
+		<thead>
+			<tr>
+				<th>Competencies</th>
+				<th>Job Families</th>
+                <th>Roles</th>
+			</tr>
+		</thead>
+		<tbody>
+        <td>
+			<tr class="myRow" id="pop-over" *ngFor="let role of roles">
+				<td>{{ role.name }}</td>
+			</tr>
+        </td>
+        <td>
+			<tr class="myRow" id="pop-over" *ngFor="let role of roles">
+				<button class="button1">{{ role.name }}</button>
+			</tr>
+        </td>
+        <td>
+			<tr class="myRow" id="pop-over" *ngFor="let role of roles">
+				<button class="button1">{{ role.name }}</button>
+			</tr>
+        </td>
+		</tbody>
+	</table>
+  </article>
+</section>
+<!--
+<table style="width: 100%">
+    <td style="width: 30%">
+        <h1>Competencies:</h1>
+        <div style="text-align: center; background-color: white; overflow: auto; border: 1px solid black; height: 300px; width: 100%; float: left;">Competencies <br></div>
+
+    </td>
+    <td style="width: 30%">
+        <h1>Job Families:</h1>
+        <div style="text-align: center; background-color: white; overflow: auto; border: 1px solid black; height: 300px; width: 100%; float: left;">Job Families <br></div>
+    </td>
+    <td align="middle" style="width: 40%;">
+        <h1>Roles:</h1>
+        <div style="text-align: center; background-color: white; overflow: auto; border: 1px solid black; height: 300px; width: 100%; float: middle;">Roles <br>
+        <tr float="middle" class="myRow" id="pop-over" *ngFor="let role of roles">
+            <button class="button1">{{ role.name }}</button>
+        </tr>
+        </div>
+    </td>
+</table> -->
```

Comment:
Are we going to be using this? If not we should remove

Reply:
@ShaneKainos